app.sh

```diff
@@ -37,6 +38,10 @@ if [ $# -gt 0 ]; then
         echo "Updating conda environment"
         conda env update -p "$PREFIX" --file environment.yml
         ;;
+      -t | --test )
+        echo "Running test suite"
+        python -m unittest discover -s ./app/main/tests/
+        ;;
       -e | --export )
```

Comment:
Not sure where the best place is for this comment, but I'll put it here as it's generic.

The additions to the `app.sh` are perfect, but I have a few comments:
1. RE the location of the tests. I'd like to see it under `./app/tests`. The main package is meant for the application source and I don't think tests should be a part of it.
2. For running the tests, I'd like to add a `test` command to the `manage.py`. You'll notice I have a TODO there to add it. Check out the implementation [here](https://github.com/cosmic-byte/flask-restplus-boilerplate/blob/master/manage.py#L28). Then for running the test via the `app.sh` file, we can simply run `python manage.py test`.
3. Let's force a pattern for naming the files for the tests to be prefixed with `test_*`. Let me know if you disagree.